### ISSUE TITLE
[Feature Request] Remember tasks and subtasks toggled state

### ISSUE THREAD

**Issue by Natetronn** [mrvladus/Errands#105]
tl;dr Please add a third option for the toggle state of Tasks and Subtasks: Open, closed and the new one, leave everything the way it was before we closed the app. The all or nothing is less than ideal for one's workflow.

Currently, we have the setting to "Expand Sub-Tasks on Startup" and turning that off is a great start (or on if you prefer that.)

However, it would be nice for it to remember the actual state of our tasks and subtasks toggle state (opened or closed) and return us right back where we left off after closing the app. If they are open, start them opened and if they are closed, start them closed, this on a task by task and subtask by subtask basis. This way our previous focus isn't lost and we'll do less clicking in the long run.

**Comment by mrvladus** [comment]
Good idea! I'll add this in the next major version.
